features/bootstrap/FeatureContext.php

```diff
@@ -29,7 +30,7 @@ class FeatureContext implements Context, SnippetAcceptingContext
      */
     public function __construct()
     {
-        $this->dockingStations = new \Cocoders\InMemory\CityBike\DockingStations();
+        $this->dockingStations = new \Cocoders\File\CityBike\DockingStationsFileRepo();
```

Comment:
@p8pepe You should use InMemory here - behat suite will be faster then probably :)